docs/migrate/migrate-e2e.rst

```diff
@@ -0,0 +1,258 @@
+End to end example
+==================
+
+As a comprehensive example, we will walk through a common use case that
+would require the user to migrate code to use the Qiskit Runtime
+execution model. 
+
+Overview
+--------
+
+Recall that when choosing which primitive to use, we first need to
+understand whether our algorithm needs to generate the entire quasi-probability
+distribution, or expectation values of observables. 
+
+In our use case, an iterative amplitude estimation (IAE), we want to
+discover one of the probability amplitudes of a quantum state. Thus, the
+algorithm should rely on the Sampler primitive.
+
+.. _amplitude:
+
+Amplitude estimation use case
+-----------------------------
+
+In this example, we look at the case that uses IAE approach to solve an
+estimation problem. Historically, on IBM Quantum Platform, users could
+achieve this by configuring epsilon and alpha parameters on the
+``QuantumInstance`` object, as shown in the code example without
```

Comment:
But the code doesn't pass epsilon and alpha to `QuantumInstance`. They are passed to `IterativeAmplitudeEstimation`